ale/base/data_naif.py

```diff
@@ -109,7 +109,7 @@ def reference_frame(self):
     @property
     def _sun_position(self):
         sun_state, _ = spice.spkezr("SUN",
-                                     self.center_ephemeris_time,
+                                     self.center_time,
```

Comment:
Do we need `center_time` as part of the base Driver interface?

Reply:
Doesn't CSM use center time as acquisition time for framers?

Reply:
It does, we have it in the Framer mix-in right now, should it be across all of the sensor_types? It looks like yes.